ipfx/x_to_nwb/DatConverter.py

```diff
@@ -183,6 +182,25 @@ def _convertTimestamp(self, heka_elapsed_seconds):
     def _isValidAmplifierState(ampState):
         return len(ampState.StateVersion) > 0
 
+    @staticmethod
+    def _getAmplifierState(bundle, series, trace_index):
+        """ Return the amplifier state object taking into account different PatchMaster versions """
```

Comment:
Could you please add a more detailed docstring (describe parameters, returns) to this function and perhaps some other functions to help better understand them.